openfe/setup/errors.py

```diff
@@ -0,0 +1,3 @@
+ABSTRACT_ERROR_STRING = ("'{cls}' is an abstract class and should not be "
```

Comment:
Do we envision having abstract classes outside of setup? If so maybe it needs to be in the top level directory?

Reply:
I had the samne thought. So far, I'm operating under the assumption that we are likely to split into multiple repos (something we should discuss on Monday, probably).

If multiple repos, obviously it must be here. In single repo, then I agree that it belongs top level.

Reply:
How about a temporary TODO so we get back to it later? (not sure how folks feel about this, there are fun gh bots that will scrape all the TODO comments and turn them into issues for you)

(feel free to say no and merge tbh, it's not like we're likely to forget either)

```suggestion
# TODO: move if we don't split to multiple repos
ABSTRACT_ERROR_STRING = ("'{cls}' is an abstract class and should not be "
```

Reply:
I agree with the principle of leaving a TODO on this sort of thing, but in this specific case, I'm hoping we'll decide about whether to split repos very soon -- so then it'll be decided soon enough that we don't need the reminder. And then if we need to move the file, it'll be a quick PR right after the meeting! 😊